network/src/connectivity_manager/mod.rs

```diff
@@ -493,6 +506,16 @@ where
         // Dial peers which are eligible but are neither connected nor queued for dialing in the
         // future.
         self.dial_eligible_peers(pending_dials).await;
+
+        // Sample final state once a minute
+        sample!(
+            SampleRate::Duration(Duration::from_secs(60)),
```

Comment:
should this be in peer manager instead? or even a dedicated unit that just listens to all incoming events and sets up a dedicated 60 second callback?

Reply:
I put it here, so we wouldn't have to deal with logging it on a non-permissioned network.  But, maybe we want that?

Reply:
Done.